src/app/wallet/wallet/shared/transaction-table/transaction-table.component.html

```diff
@@ -83,10 +83,15 @@
         <div fxFlex="1 0 155px" *ngIf="display.amount" class="tx-value">
           <span class="amount" *ngIf="!tx.requires_unlock"
                 [ngClass]="{'positive': tx.getAmountObject().getAmount() > 0, 'negative': tx.getAmountObject().getAmount() < 0 }">
-            <span class="big number">{{ tx.getAmountObject().getIntegerPartAsString() }}</span><!-- inline element comment hack
-            --><span class="point">{{ tx.getAmountObject().dot() }}</span><!--
-            --><small class="small number">{{ tx.getAmountObject().getFractionalPart() || '' }}</small>
-            <span class="currency">GHOST</span>
+                <span *ngIf="tx.getCategory() === 'stake' && tx.getAmountObject().getAmount() === 0">
+                  POOL STAKE
+                </span>
+                <span *ngIf="tx.getCategory() !== 'stake' || tx.getAmountObject().getAmount() !== 0">
```

Comment:
Dont understand this line here,explain why this checks if category isnt stake and amount isnt 0

Reply:
it's for the other transactions, like internal transfer, etc. let me change it with else

Reply:
I dont see the need for this extra check,why not use else?
or rather add a func in the code instead of adding a check here.ie add code to the ts file to check if the tx is a pool stake.else etc.